docs/hub/spaces.md

```diff
@@ -28,7 +28,7 @@ Each Spaces environment is limited to 16GB RAM and 8 CPU cores. Organization [su
 
 ## Streamlit and Gradio
 
-Spaces support [Streamlit](https://streamlit.io/) and [Gradio](https://gradio.app/) for quickly building apps in Python. The default environment runs version `"1.0.0"` of Streamlit and the latest version of Gradio. We recommend you try both because they're really awesome! Here are some of our thoughts on Streamlit and Gradio:
+Spaces support [Streamlit](https://streamlit.io/) and [Gradio](https://gradio.app/) for quickly building apps in Python. The default environment runs whichever latest version of Streamlit and Gradio is the current one at the time you create your Space repo. We recommend you try both because they're really awesome! Here are some of our thoughts on Streamlit and Gradio:
```

Comment:
do they get set at the time of creating the repo, or whenever a new docker image is built (which can be after every commit?). Also, can they set the version if they have a specific requirement?

(nit: wrapping the lines around instead of a single line paragraph would also be nice :P )

Reply:
set at repo creation

and users can override it with whatever version is supported (i think this is already documented, @cbensimon?)

Reply:
Yes, this is exact. And yes, already documented but only for Streamlit currently. I'll update it